<launch>

    <node pkg="front_wheel_center_tf" type="add_stanley_tf" name="add_stanley_tf"/>

    <node pkg="tf_listener_test" type="tf_listen" name="tf_listen"/>

    <node pkg="my_car_waypoint_display" type="my_car_waypoint_publish" name="my_car_waypoint_publish"/>
    
    <!-- <node pkg="my_car_stanley_method" type="my_car_stanley_method" name="my_car_stanley_method"/> -->

</launch>